### ISSUE TITLE
Lunch bot should be able to count diners

### ISSUE THREAD

**Issue by the-mikedavis** [tonyvanriet/lunch-bot#4]
Once you choose a restaurant, you should be able to specify a number of diners:

```
@lunchbot choose shake shack
@lunchbot today 12
```

Then in the `today/0` command, in addition to showing who's **in**, lunch-bot should show how many people are in **of** the number of specified diners:

```
today
;=> Ordering Shake Shack
;=> ...
;=> 10 of 12 peolple are **in**
```

**Comment by tonyvanriet** [comment]
sounds great. PR plox

**Comment by the-mikedavis** [comment]
whoops tagged the wrong issue in the PR description. closed by #5.
